met-web/src/components/publicDashboard/DashboardContext.tsx

```diff
@@ -40,6 +43,8 @@ export const DashboardContextProvider = ({ children }: DashboardContextProviderP
     const [engagement, setEngagement] = useState<Engagement>(createDefaultEngagement());
     const [isEngagementLoading, setEngagementLoading] = useState(true);
 
+    const dashboardType = dashboardTypeParam ? dashboardTypeParam : 'public';
```

Comment:
all places where you use a string, always use it through a variable. and the same variable preferably. i.e. let's not use 'public' directly but use variable PUBLIC